### DISCUSSION TITLE
Relationship advice and mental health counselling service built in-app

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#14] [Ideas & Features]
One idea from a meeting was to add an option to assist people that are struggling with mental health challenges or self-image issues related to the use of the app. There could be a part of the app, or at least a link within the app, to some sort of service option.

**Comment by Oshyan** [comment]
This raises serious challenges of liability and other issues. It's a huge can of worms. I think I understand the desire to maybe have something like this, but I don't think we should consider it in-scope for quite a while, if at all.

**Comment by ajbott** [comment]
I can see Oshyan's concern, but I think this is definitely something worth looking into. For people that are seriously trying to find a date any the app is still failing them, I'd like to at least be able to listen and investigate why the app isn't working for them. We could offer help in tips on how to use the app better through the data we've learned for what works, help with their profile, or even their messaging techniques. Later on we could help out by offering free 3rd party therapy services for the users to help in their dating life.
